packages/hadron-build/signtool/install.js

```diff
@@ -18,5 +19,16 @@ if (!fs.existsSync(originalSigntool)) {
   throw new Error('Original signtool.exe to be replaced not found');
 }
 
+const buildScriptDir = __dirname;
```

Comment:
Adding a postinstall script to build signtool is 1) unnecessary for local development and 2) adds a dependency on the correct Go version to local dev environments.  It's also not necessary for the majority of CI.

Is it possible to just build signtool in CI when necessary instead of always building it?

Reply:
yeah that's a good idea. only build it when we need to sign something.